src/ResourceManager/Network/Commands.Network/Cortex/ExpressRouteGateway/ExpressRouteConnection/GetAzureRmExpressRouteConnectionCommand.cs

```diff
@@ -49,7 +49,7 @@ public class GetAzureRmExpressRouteConnectionCommand : ExpressRouteConnectionBas
             ParameterSetName = CortexParameterSetNames.ByExpressRouteGatewayName,
             HelpMessage = "The parent resource name.")]
         [ValidateNotNullOrEmpty]
-        public string ParentResourceName { get; set; }
+        public string ParentExpressRouteGatewayName { get; set; }
```

Comment:
Most cmdlets in PS don't use "Parent" prefix, could you please use "ExpressRouteGatewayName"?
@cormacpayne @maddieclayton could you please confirm?